src/main/java/frc/robot/RobotContainer.java

```diff
@@ -139,16 +142,22 @@ private void configureBindings() {
     new Trigger(m_exampleSubsystem::exampleCondition)
         .onTrue(new ExampleCommand(m_exampleSubsystem));
 
+    // new Trigger(driverController::getCrossButton).onTrue(new autoAimParallel(driveTrain/*, shooter*/));
+    new Trigger(driverController::getPSButton).onTrue(new InstantCommand(driveTrain::zeroGyroscope));
+    SmartDashboard.putData(new RotateRobot(driveTrain, driveTrain::calculateSpeakerAngle));
+    new Trigger(driverController::getCrossButton).onTrue(new InstantCommand(()->SmartDashboard.putNumber("calculated robot angle", driveTrain.calculateSpeakerAngle())));
+    // new Trigger(driverController::getCrossButton).onTrue(new autoAimParallel(driveTrain));
+    new Trigger(driverController::getCrossButton).onTrue(new RotateRobot(driveTrain, driveTrain::calculateSpeakerAngle));
```

Comment:
It might not be a good practice to have multiple triggers bound to the same button, maybe combine them to [on cross button].onTrue(printAngle).AndThen(drivetrain:moveToAngle)